pkg/resources/database.go

```diff
@@ -142,6 +216,22 @@ func createDatabaseFromReplica(d *schema.ResourceData, meta interface{}) error {
 	return ReadDatabase(d, meta)
 }
 
+func createDatabaseFromReplicaConfig(d *schema.ResourceData, meta interface{}) error {
```

Comment:
this looks exactly like createDatabaseFromReplica(). if all you are doing is concatenating the organization_name account_name and primary_db_name, i dont know that this adds much value compared to using the from_replica argument. i am wondering if this can instead be an output of the database resource that has replication_configuration enabled. e.g. "database_identifier". Another option is create a datasource to construct identifiers based on input arguments.